.travis.yml

```diff
@@ -27,6 +21,14 @@ install:
 script:
   - $TRAVIS_BUILD_DIR/sysconfcpus/bin/sysconfcpus -n 2 npm run make
   - $TRAVIS_BUILD_DIR/sysconfcpus/bin/sysconfcpus -n 2 npm test
+  # Test in several timezones (positive, negative, zero offset; DST; partial-hour offsets)
```

Comment:
if these are just environment variables they can turn into a build matrix https://docs.travis-ci.com/user/customizing-the-build/#Build-Matrix